docs(self-hosted config): use admonitions (#13888)
renovatebot/renovate: docs/usage/self-hosted-configuration.md

If this is set to `myprefix_` the final container created from `renovate/node` image would be named `myprefix_node` instead of currently used `renovate_node` and be labeled `myprefix_child` instead of `renovate_child`.

<!-- prettier-ignore -->
!!! note

>     Dangling containers will not be removed until Renovate is run with the same prefix again.

Author: ```suggestion
    Dangling containers will only be removed when Renovate runs again with the same prefix.
```

How about something like this?
